krod/core/memory/__init_.py

```
@@ -0,0 +1,7 @@
+from .conversation_memory import ConversationMemory
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)